velox/functions/lib/MapFromEntries.cpp

```diff
@@ -208,26 +211,36 @@ class MapFromEntriesFunction : public exec::VectorFunction {
           nulls, indices, decodedRowVector->size(), rowVector->childAt(1));
```

Comment:
For Spark, given the result is NULL for NULL input, do we need to set NULL for the keys and values? Actually I find the nulls buffer seems to be mixed between the map vector and key/value vector.